src/cluster_legacy.c

```diff
@@ -5242,12 +5242,12 @@ void clusterSetMaster(clusterNode *n) {
  * Nodes to string representation functions.
  * -------------------------------------------------------------------------- */
 
-struct redisNodeFlags {
+struct serverNodeFlags {
```

Comment:
```suggestion
struct clusterNodeFlags {
```
Server doesn't really make sense here, since they're cluster node flags.